ape_trezor/client.py

```diff
@@ -0,0 +1,119 @@
+from typing import Optional, Tuple
+
+from ape.api.accounts import TransactionAPI
+from eth_typing.evm import ChecksumAddress
+from trezorlib import ethereum
+from trezorlib.client import get_default_client  # type: ignore
+from trezorlib.exceptions import PinException, TrezorFailure
+from trezorlib.tools import parse_path as parse_hdpath  # type: ignore
+from trezorlib.transport import TransportException
+
+from ape_trezor.exceptions import TrezorAccountException, TrezorClientError
+from ape_trezor.hdpath import HDBasePath, HDPath
+
+
+class TrezorClient:
+    """
+    This class is a client for the Trezor device.
+    """
+
+    def __init__(self, hd_root_path: HDBasePath):
+        try:
+            self._client = get_default_client()
+        except TransportException as exc:
+            message = (
```

Comment:
This exception message is used twice (once again on line 71), it _could_ be abstracted into a separate custom exception